api/src/glfs.c

```diff
@@ -207,7 +207,7 @@ create_master(struct glfs *fs)
         goto err;
     }
 
-    fs->ctx->master = master;
+    fs->ctx->primary = master;
```

Comment:
I think the usage of master should be changed here as well and in the routine.

Reply:
I am sending changes for "api" folder on top of this patch.  Can we take this patch in independent of that change?